### PR TITLE
Websocket example using the WebInterface

### PR DESCRIPTION
**Pull request by dariusc93**
This is a simple example to just show how to use websockets with the web interface.

### CODE REVIEW

File: examples/web_websocket/dub.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿{
+	"name": "web-websockets-example",
+	"description": "Example for using the WebSocket feature with WebInterface",
+	"authors": [ "Sönke Ludwig" ],
```
Comment (schuetzm) [inline examples/web_websocket/dub.json]: You aren't Sönke, are you?

**Comment by s-ludwig** [comment]
Thanks, thats definitely good to have! Generally looks good to merge.

Apart from the author field, it would be good to get dub.json and websocket.js a trailing newline as many tools complain about this (even if it's not really relevent for D).

**Comment by dariusc93** [comment]
I do see that the websocket.js needs one but the dub.json doesnt need one, unless git is removing it when its commited.

**Comment by s-ludwig** [comment]
Okay, thanks! (You added the dub.json newline in 85e4811b11ec52276d706dff97c0015ea6974a6f)

**Pull request merged**